src/coreclr/jit/morph.cpp

```diff
@@ -4619,7 +4897,16 @@ GenTree* Compiler::fgMorphMultiregStructArg(GenTree* arg, fgArgTabEntry* fgEntry
             //
             lvaSetVarDoNotEnregister(varNum DEBUG_ARG(DoNotEnregisterReason::LocalField));
         }
-#endif // TARGET_ARM
+#elif defined(TARGET_LOONGARCH64)
```

Comment:
For now it looks like LA could use the TARGET_ARM64 code above since LA currently has struct promotion disabled. Is there a reason not to?

Reply:
Thanks